src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +104,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(new CandidateValidator())
+                .sorted(new CandidateValidator.CandidateComparator())
```

Comment:
```suggestion
                .sorted()
```
try without comparator